frontend/src/components/Spiner.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Spin } from 'antd';
+import styled from "styled-components";
+import { LoadingOutlined } from '@ant-design/icons';
+const antIcon = <LoadingOutlined style={{ fontSize: 24 }} spin />;
+
+const SpinerStyle = styled.div`
+position: fixed;
+z-index: 999;
+height: 4em;
+// width: 4em;
+overflow: visible;
+margin: auto;
+top: 0;
+left: 50%;
+bottom: 0;
+right: 0;
+`;
+
+export const CustomSpinner = ({
```

Comment:
Can we have just Spinner component and pass props for different styles?

Also cant we just use Antd approach: https://codesandbox.io/s/ye0e6

Reply:
@jyash97 is like boolean props name as **_defaultSpiner_** type as **_boolean_** .Based boolean we have to render the Spinner indicator right yash?